content/_policy/contact._en.md

```diff
@@ -29,7 +29,7 @@ help_center_content: >-
 partner_content: >-
   ## Partner with Login.gov
 
-  Interested in using Login.gov at your agency? Please [visit our partners website](https://partners.login.gov/) or [contact us](https://share.hsforms.com/16DIoo--rTU2xbNW1MShkBg3ak9e).
+  Interested in using Login.gov at your agency? Please [visit our partners website](https://partners.login.gov/) or [contact us](https://login.gov/partners/contact/).
```

Comment:
Could we use a root-relative path so that it links when running the site in local development? (Ideally this would also work in Federalist, but I don't know that we support it currently for this sort of Markdown-based content)

```suggestion
  Interested in using Login.gov at your agency? Please [visit our partners website](https://partners.login.gov/) or [contact us](/partners/contact/).
```

Reply:
I think [our baseurl plugin](https://github.com/18F/identity-site/blob/main/_plugins/base_url.rb) works for markdown'ed content

Reply:
I was also going to say I've used `link` before in the dev docs: https://github.com/18F/identity-dev-docs/blob/main/_pages/testing.md?plain=1#L25

Reply:
> I think [our baseurl plugin](https://github.com/18F/identity-site/blob/main/_plugins/base_url.rb) works for markdown'ed content

Ooh, yes, I thought we'd had something like that, but I wasn't able to find it when looking earlier. So it should "just work" when using the root-relative path, then.